dts/arm/atmel/same70.dtsi

```diff
@@ -12,6 +12,8 @@
 / {
 	aliases {
 		watchdog0 = &wdt;
+
+		eth-0 = &gmac;
```

Comment:
Most of the other aliases do not seem to have `-` in it, so why just not have `eth0` instead?

Reply:
Other SAM series devices are using the `-` convention and this is a deliberate attempt to align with them:
https://github.com/zephyrproject-rtos/zephyr/blob/ab15602b796455ca284abf7234280e6b0ee5e032/dts/arm/atmel/samd20.dtsi#L10-L12

https://github.com/zephyrproject-rtos/zephyr/blob/ab15602b796455ca284abf7234280e6b0ee5e032/dts/arm/atmel/samd5x.dtsi#L23-L25

Reply:
If we were to follow the Linux convention, this should really be `ethernet0`.

What do you think? @galak

Reply:
does the alias actually work??

Reply:
> does the alias actually work??

@galak Yes

on `sam_e70_xplained` that has max queue count of 3:
https://github.com/zephyrproject-rtos/zephyr/blob/17a932c1077709f163ed0d188e257f4e65939508/dts/arm/atmel/same70.dtsi#L320

![image](https://user-images.githubusercontent.com/11345418/78987930-6a656080-7b6a-11ea-9eac-3eaa418336dc.png)